### DISCUSSION TITLE
FastAPI Wrapper

### DISCUSSION THREAD

**Discussion by matthewdeaves** [microsoft/markitdown#201] [Show and tell]
Hello,

I have been working on a FastAPI wrapper for MarkItDown, for use in my  personal projects. I have been working alongside CoPilot etc to help me build it, the repo is:

https://github.com/matthewdeaves/markitlikeitshot

I'm still refining and adding features to it, such as a separate admin API to provide the same user and api key management that the CLI interface offers.

Would be good to hear if anyone else was thinking the same or has ideas for it!

**Comment by pp-researcher** [comment]
I believe Certificate implementing based authentication for the client would be a good idea.
